--- config.h	2015-07-11 23:11:18.602482167 +0900
+++ config.h	2015-07-11 23:12:41.785501334 +0900
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=false:autohint=false";
-static int borderpx = 2;
+static char font[] = "M+ 1mn:pixelsize=14";
+static int borderpx = 1;
 static char shell[] = "/bin/sh";
 static char *utmp = NULL;
 static char stty_args[] = "stty raw -echo -iexten echonl";
@@ -15,7 +15,7 @@
 static char vtiden[] = "\033[?6c";
 
 /* Kerning / character bounding-box multipliers */
-static float cwscale = 1.0;
+static float cwscale = 0.5;
 static float chscale = 1.0;
 
 /*
@@ -96,9 +96,9 @@
  * Default colors (colorname index)
  * foreground, background, cursor
  */
-static unsigned int defaultfg = 7;
-static unsigned int defaultbg = 0;
-static unsigned int defaultcs = 256;
+static unsigned int defaultfg = 258;
+static unsigned int defaultbg = 263;
+static unsigned int defaultcs = 259;
 
 /*
  * Colors used, when the specific fg == defaultfg. So in reverse mode this
