diff -ur st/config.h st.patched/config.h
--- st/config.h	2015-01-11 01:22:16.000000000 +0900
+++ st.patched/config.h	2015-06-05 02:03:54.275220683 +0900
@@ -5,12 +5,12 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=false:autohint=false";
+static char font[] = "Input:pixelsize=16";
 static int borderpx = 2;
 static char shell[] = "/bin/sh";
 
 /* Kerning / character bounding-box mutlipliers */
-float cwscale = 1.0;
+float cwscale = 0.5;
 float chscale = 1.0;
 
 /*
@@ -51,6 +51,16 @@
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
+	/* 8 bright colors */
+	"white",
+	"red",
+	"green",
+	"yellow",
+	"#5c5cff",
+	"magenta",
+	"cyan",
+	"gray50",
+
 	/* 8 normal colors */
 	"black",
 	"red3",
@@ -61,16 +71,6 @@
 	"cyan3",
 	"gray90",
 
-	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
-
 	[255] = 0,
 
 	/* more colors can be added after 255 to use with DefaultXX */
